### PR TITLE
DOCSP-24855 adds Stable API options

### PR DESCRIPTION
**Pull request by jmd-mongo**
# STAGING

[Stable API Options](https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCSP-24855/reference/options/#stable-api-options)

# JIRA

https://jira.mongodb.org/browse/DOCSP-24855

### CODE REVIEW

File: source/reference/options.txt [hunk 1]
```diff
@@ -76,6 +76,34 @@ General Options
...
+   currently the only supported value.
+
+.. option:: --apiStrict
+
+   Specifies that the server will respond with :ref:`APIStrictError 
+   <api-strict-resp>` if your application uses a command or behavior 
+   outside of the :ref:`Stable API <stable-api>`.
+   
+   If you specify :option:`--apiStrict`, you must also specify
+   :option:`--apiVersion`.
```
Comment (sarah-olson-mongodb) [inline source/reference/options.txt:96]: (question) Is this for apiStrict? From the description, it seems at least also intended for apiDeprecationErrors, but perhaps only intended?

File: source/reference/options.txt [hunk 2]
```diff
@@ -76,6 +76,34 @@ General Options
...
+
+.. option:: --apiDeprecationErrors
+
+   Specifies that the server will respond with :ref:`APIDeprecationError
+   <api-deprecation-resp>` if your application uses a command or behavior 
+   that is deprecated in the specified :ref:`apiVersion
```
Comment (sarah-olson-mongodb) [inline source/reference/options.txt:102]: (linking)
This is the second instance of the apiVersion link. Fine to omit.

**Review by sarah-olson-mongodb — APPROVED** [review]
LGTM.
Thanks for this work Joe!
I left two items for your consideration.

File: source/reference/options.txt (hunk 1 above)
Comment (jmd-mongo) [reply source/reference/options.txt:96]: Good eye! I've noted this in the description for --apiDeprecationErrors as well.

File: source/reference/options.txt (hunk 2 above)
Comment (jmd-mongo) [reply source/reference/options.txt]: Omitted.

File: source/reference/options.txt [hunk 3]
```diff
@@ -76,6 +76,36 @@ General Options
...
+.. option:: --apiVersion <version number>
+
+   Specifies the :ref:`apiVersion <api-version-desc>`. ``"1"`` is 
+   currently the only supported value.
+
+.. option:: --apiStrict
+
+   Specifies that the server will respond with :ref:`APIStrictError 
+   <api-strict-resp>` if your application uses a command or behavior 
+   outside of the :ref:`Stable API <stable-api>`.
```
Comment (Anemy) [inline source/reference/options.txt:93]: Should we mention that when this is not specified then mongosh defaults to a non-strict client? I think it's probably fine as is as well.

**Review by Anemy — APPROVED** [review]
lgtm - left one thought/suggestion.

**Pull request merged**